.gitlab/ci/.gitlab-ci.sdk-nightly.yml

```diff
@@ -180,52 +273,69 @@ demisto-sdk-nightly:run-commands-against-instance:
       echo "Server URL: $DEMISTO_BASE_URL"
-    - section_start "Unlock HelloWorld Integration and Playbook" --collapsed
-    - python3 Tests/sdknightly/unlock_entity.py integration "HelloWorld,HelloWorld Feed"
-    - section_end "Unlock HelloWorld Integration and Playbook"
+
     - section_start "Create and Upload Entities to XSOAR Instance" --collapsed
     - python3 Tests/sdknightly/create_entities_for_nightly_sdk.py HelloWorld --artifacts-folder $ARTIFACTS_FOLDER
+    - python3 Tests/sdknightly/create_entities_for_nightly_sdk.py AzureSentinel --artifacts-folder $ARTIFACTS_FOLDER
+    - demisto-sdk upload -i Packs/AzureSentinel -z --insecure
     - *upload-entities-to-cortex-xsoar
     - section_end "Create and Upload Entities to XSOAR Instance"
+
     - section_start "Download Entities from XSOAR Instance" --collapsed
-    - demisto-sdk download -o Packs/HelloWorld/ -i HelloWorld --insecure -f
+    - demisto-sdk download -lf --insecure
+    - demisto-sdk download -o Packs/HelloWorld/ -i CommonServerUserPython --insecure -f
     - demisto-sdk download -o Packs/HelloWorld/ -i HelloWorld-Test --insecure -f
-    - demisto-sdk download -o Packs/HelloWorld/ -i "Hello World IncidentField Test" --insecure -f
-    - demisto-sdk download -o Packs/HelloWorld/ -i "Hello World Alert Test" --insecure -f
-    - demisto-sdk download -o Packs/HelloWorld/ -i "Hello World Test - Incoming Mapper" --insecure -f
     - section_end "Download Entities from XSOAR Instance"
-    - section_start "Format Content Items" --collapsed
-    - demisto-sdk format -i Packs/HelloWorld/ -nv -ud -y
-    - section_end "Format Content Items"
-    - section_start "Run Validate" --collapsed
-    - demisto-sdk validate -i Packs/HelloWorld/
-    - section_end "Run Validate"
-    - section_start "Run Lint" --collapsed
-    - demisto-sdk lint --console-log-threshold DEBUG -i Packs/HelloWorld/
-    - section_end "Run Lint"
-    - *change-file-ids
-    - *upload-entities-to-cortex-xsoar
-    - section_start "Destroy instances"
-    - python3 ./Tests/scripts/destroy_instances.py --artifacts-dir $ARTIFACTS_FOLDER --env-file $ARTIFACTS_FOLDER/env_results.json --instance-role "$INSTANCE_ROLE"
+
+    - section_start "Install Packs and Test-Module" --collapsed
+    - cat "$ARTIFACTS_FOLDER/filter_file.txt" 
+    - ./Tests/scripts/install_content_and_test_integrations.sh || EXIT_CODE=$?
+    - cp -f "${ARTIFACTS_FOLDER}/conf.json" Tests/conf.json
+    - section_end "Install Packs and Test-Module"
+
+    - section_start "Wait Until Server Ready"  --collapsed
+    - echo Going to sleep for 15 minutes to allow server finish indexing
+    - sleep-with-progress 900 30 "Sleeping... " 150
+    - echo "Done sleeping!"
+    - section_end "Wait Until Server Ready"
+
+    - !reference [.run_tests]
+
+    - job-done
+    - exit $EXIT_CODE
+  after_script:
+    - !reference [.default-after-script]
+    - !reference [.install_ssh_keys]
+    - !reference [.ssh-config-setup]
```

Comment:
do we need it in the after script?

Reply:
Yes, as the `after script` section does not have the credentials of the machine from the script